app/modules/asset_groups/models.py

```diff
@@ -118,6 +119,50 @@ class AssetGroupSighting(db.Model, HoustonModel):
     # May have multiple jobs outstanding, store as Json obj uuid_str is key, In_progress Bool is value
     jobs = db.Column(db.JSON, nullable=True)
 
+    def start_sage_detection(self, model_config, detection):
+        # TODO make this a task so that it's non blocking but handle the response and
+        # inform AssetGroupSighting if it fails
+        current_app.acm.request_passthrough_result(
+            'job.detect_request', 'post', {'params': model_config}, detection
+        )
+
+    def run_sage_detection(self, model):
+        job_id = uuid.uuid4()
+        base_url = current_app.config.get('BASE_URL')
+        callback_url = f'{base_url}api/v1/asset_group/sighting/{str(self.guid)}/sage_detected/{str(job_id)}'
+        # TODO use model to build up the input, also not clear on where the endpoint & function come from
```

Comment:
yeah i think this is currently under development.  likely @drewblount will get this into the ia.json stuff.

Reply:
That's the intention. This is the startpoint for @drewblount to build on.